packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -234,24 +146,29 @@ export const makeZCFZygote = (
   ) => {
     assertUniqueKeyword(keyword);
 
-    const zoeMintP = E(zoeInstanceAdmin).makeZoeMint(
+    // TODO is this AWAIT ok? Or does this need atomicity?
+    // We probably need the same keyword reservation logic that
+    // was added to addPool
```

Comment:
Need to think about this interleaving hazard. See extended comment on the one in zcfMint.js

Reply:
@erights and I discussed the original issue here and decided there's no problem.